_infra/helm/sample/templates/cronjob.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: sample-export
+spec:
+  schedule: "{{ .Values.export.cron }}"
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: curl
+            image: curlimages/curl:7.71.0
+            command: ["/bin/sh", "-c"]
+            args:
+              - |
+                curl -X POST "http://{{ .Chart.Name }}:{{ .Values.dns.wellKnownPort }}/samples/export" -H "accept: application/json"
```

Comment:
One very small nit pick.  Can you add the -s and -S flags to this like in https://github.com/ONSdigital/rm-collection-exercise-service/blob/master/_infra/helm/collection-exercise/templates/distribute-sample-unit-cronjob.yaml#L37

The reason for this is that you get a bunch of 
```
0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0
```
and 
```
% Total % Received % Xferd Average Speed Time Time Time Current
```
 'error' logs when you don't suppress the output